README.md

```diff
@@ -52,6 +52,13 @@ Download the pre-compiled package for your operating system.  Install required d
 
   where `$mincToolsDirectory` is the path where the MINC toolkit is installed (e.g. `/opt/minc/` OR `/opt/minc/$mincToolsVersion/` for more recent installs)
 
+For the defacing scripts, you will also need to download the pre-compiled `bic-mni-models` and `beast` data and model packages for you operation system.
+
+   ```bash
+   sudo dpkg -i bic-mni-models-0.1.1-20120421.deb
+   sudo dpkg -i beast-library-1.1.0-20121212.deb
```

Comment:
Do these package need the exact version number?
If not suggest using ```sudo apt-get install bic-mni-models``` and ```sudo apt-get install beast-library``` 
These command will fetch the latest packages (from a repo).  Even if they don't get updated often
Do these package are on neuro-debian?

Reply:
oups... I should have removed the actual version and put it in `<version>` as for the MINC tool. Good catch!

Since we tell the users to actually go download the tools from the MINC toolkit page a few lines above and those packages are on the same page, will keep the rest of the instruction as is. Note that I am not sure that the MINC tools you can get using `apt-get` are necessarily the latest version (last time I checked it was outdated with a few bugs which is why we now recommend downloading it directly from the MINC tools webpage).